nexus/db-model/src/deployment.rs

```diff
@@ -467,6 +467,8 @@ pub struct BpOmicronZone {
     pub snat_last_port: Option<SqlU16>,
 
     disposition: DbBpZoneDisposition,
+    expunged_as_of_generation: Option<Generation>,
```

Comment:
nit: Since these are actually fields of disposition, it may be helpful to prefix them with `disposition_`. I know it makes them quite long, but it may add clarity.

Reply:
Agreed, I think that's clearer. Added the prefix in [db4ca5c](https://github.com/oxidecomputer/omicron/pull/7558/commits/db4ca5cd168b47fcf8087488e41e102e02863801) (except to the fields of `DbBpZoneDispositionColumns`, where it's already clear they're all related to the disposition)